src/types.ts

```diff
@@ -112,6 +123,37 @@ export interface PaletteColor {
   color: string | null;
 }
 
+interface TextAlign {
+  auto: boolean;
+  align: string;
+}
+
+export interface ExtendedNxAttrExprInfo extends EngineAPI.INxAttrExprInfo {
+  id: string;
+  qMin: number;
+  qMax: number;
+  qContinuousAxes: boolean;
+  qIsCyclic: boolean;
+  qFallbackTitle: string;
```

Comment:
I thought these were already on here? that the only thing you needed to extend with is the id prop? https://help.qlik.com/en-US/sense-developer/May2022/Subsystems/EngineJSONAPI/Content/models-nxattrexprinfo.htm

Reply:
right, fixed